### PR TITLE
Rhel9

### PR DESCRIPTION
**Pull request by nocturnalastro**

### CODE REVIEW

**Review by novacain1 — COMMENTED** [review]
Just a suggestion on typos here, misspellings I noticed.

**Comment by nocturnalastro** [comment]
@arjuhe I opened a new PR because pushing to your branch proved a pain.

File: roles/patch_cluster/templates/patch-install-config.j2 [hunk 1]
```diff
@@ -1,8 +1,11 @@
+{# note: This current breaks the current assisted installer
```
Comment (arjuhe) [inline roles/patch_cluster/templates/patch-install-config.j2:1]: This _currently_ breaks

File: roles/setup_sushy_tools/tasks/main.yml [hunk 2]
```diff
@@ -3,8 +3,10 @@
   become: true
   block:
     - name: Install required packages
-      package:
-        name: "{{ sushy_packages }}"
+      package: # noqa: jinja[spacing]
+        name: "{{ sushy_packages | default(
+            (is_on_rhel9 | bool) | ternary(sushy_packages_rhel9, sushy_packages_rhel8)
```
Comment (arjuhe) [inline roles/setup_sushy_tools/tasks/main.yml:8]: Checking the spelling this time

File: roles/validate_inventory/defaults/main.yml [hunk 3]
```diff
@@ -20,17 +20,18 @@ supported_vendor_values:
...
-- 4.13.10
+  - 4.6.16
+  - 4.7.52
+  - 4.8.43
+  - 4.9.59
+  - 4.10.67
+  - 4.11.53
+  - 4.12.44
+  - 4.13.22
+  - 4.14.2
```
Comment (arjuhe) [inline roles/validate_inventory/defaults/main.yml:35]: Do you think the "versions tested" list from the README should match the z stream list here?

**Review by arjuhe — CHANGES_REQUESTED** [review]
Just a minor change and a question.

**Comment by nocturnalastro** [comment]
@arjuhe Could you take a look at this again :)

**Review by arjuhe — APPROVED** [review]
Thx LGTM

**Pull request merged**